diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/files/NewlinesRule.kt

```diff
@@ -342,6 +352,25 @@ class NewlinesRule(configRules: List<RulesConfig>) : DiktatRule("newlines", conf
             }
         }
 
+    private fun handleFirstValueArgument(node: ASTNode) = node
```

Comment:
It can be merged with `handleFirstValueParameter` with a couple of additional arguments

Reply:
Merged